mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
+func ListSnapshots(
+	instanceName string,
+	svc rdsiface.RDSAPI,
```

Comment:
oh, about the meaning, it meant to be `service`, but since Go uses tabs for indentation, it's pretty common to try to reduce names a lot. Similarly with `*testing.T` instance being called `t`.
This trend in Go to name thing with such short names, is something @miguelbernadi and me discussed sometimes, trying to get to a midway between Go's style, and Java too-much-verbosity.